src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/Requests/ResolveApprenticeshipOverlappingTrainingDateRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Api.Types.Requests
+{
+    public class ResolveApprenticeshipOverlappingTrainingDateRequest
+    {
+        public long? ApprenticeshipId { get; set; }
+        public long? DraftApprenticeshipId { get; set; }
```

Comment:
dont think DraftApprenticeshipId and resolutiontypes are required

Reply:
Now removed